src/common/test/componentMocking.tsx

```diff
@@ -97,14 +143,33 @@ export const renderProfileContextWrapper = async (
     return waitForDataChange();
   };
 
-  const getElement: TestTools['getElement'] = ({ testId, text }) => {
+  const getElement: TestTools['getElement'] = ({
```

Comment:
It seems that this function only wants one of the arguments per function call? Im not sure if it's easy to tell from the api of the function, that user should only pass one of them? One option could be to separate the qualifier and value. Something like

`const getElementByQualifier(qualifier, value)`. Though then we would need to type all the qualifiers, and that would also have an impact to the readability probably. 🤔

Reply:
In normal use, why would you use multiple selectors at the same time? Like id + testid, label + text or id  + text. QuerySelector with label/text/value would make sense, but others not. When selector with value is needed there is waitForElementAndValue.
Selector with label/text is not currently needed.

I added this so there is only one function to get an element.  getElement is also used in tests outside of this file. If it did not exists, all funcs from testing-library should be passed through to tests
getByLabelText(label);
getByTestId(testId);
getByText(text);
getByDisplayValue(valueSelector);
(+ create new ones for id and querySelector )

GetElement is easier to use that than finding each time the correct one from testing-library.